Server/app.js

```diff
@@ -14,7 +14,7 @@ app.get('/', (req, res) => {
 })
 
 app.get('/productID', (req, res) => {
-    var id = (req.body.productID)
+    var id = (req.query.productID)
```

Comment:
hate to be a stickler but you should be using `let `any time you aren't using `const`

Reply:
got it! force of habit ill make the changes and make another commit 👍